pkg/storage/client_raft_test.go

```diff
@@ -333,6 +333,8 @@ func TestReplicateRange(t *testing.T) {
 func TestRestoreReplicas(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
+	defer t.Skip("https://github.com/cockroachdb/cockroach/issues/40351")
```

Comment:
Why are you deferring this?

Reply:
Because I was braindead, thanks for noticing